### DISCUSSION TITLE
Mark purpur jar as executable

### DISCUSSION THREAD

**Discussion by ATMmachin3** [PurpurMC/Purpur#515] [__Dismissed__] [answered]
For the convenience of beginners, It would be a good idea to mark the purpur jar as an executable file for Ubuntu and other OS's alike. This request is kind of pointless to be fair.

Info: https://wiki.ubuntu.com/Security/ExecutableBit

**Comment by granny** [comment, marked as answer]
It's not an executable though.

**Reply by ATMmachin3** [reply]
jar files are executable through java

say as if you were not using a startup script

**Reply by BillyGalbreath** [reply]
Startup script, no startup script. Still is ran by `java -jar` and not the executable bit
